src/ansi-c/expr2c.cpp

```diff
@@ -2594,8 +2594,8 @@ std::string expr2ct::convert_code_ifthenelse(
   }
   else
     dest+=convert_code(
-        to_code(src.then_case()),
-        to_code(src.then_case()).get_statement()==ID_block ? indent : indent+2);
+        src.then_case(),
+        src.then_case().get_statement()==ID_block ? indent : indent+2);
```

Comment:
Nit pick (here and elsewhere): clang-format would like to see a bit more space. Could you please include all changes suggested by clang-format as well?